packages/block-library/src/gallery/style.scss

```diff
@@ -1,3 +1,10 @@
+.wp-block-gallery {
+
+	figcaption {
+		flex-grow: 1;
```

Comment:
This solves it, but I'd recommend moving this down into the other styles below (the ones that apply to both `.wp-block-gallery` and `.blocks-gallery-grid`). It seems weird to separate it out here, and I think it should be fine to apply it there too.

Reply:
Thanks @kjellr Add styles inside `.wp-block-gallery` and `.blocks-gallery-grid` class.